utils/augeas/Makefile

```diff
@@ -50,6 +50,9 @@ define Package/augeas-lense/description
   Set of Augeas lenses.
 endef
 
+CONFIGURE_ARGS+= \
+        --without-selinux \
```

Comment:
I would remove this backslash since it's the only argument.

Reply:
Removed, but I didn't do compile neither run test.